<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomDialog</class>
 <widget class="QDialog" name="CustomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>313</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:  blue; 
}
QFrame[accessibleName=&quot;evo-custom-dial&quot;]	
{
	/* Set this background-color for the pop-down menu in combobox */
	background-color: transparent;
	font: 16px 'Montserrat';
	color: black;
	border: 0px;
    border-radius: 20px;
}

QLabel[accessibleName=&quot;evo-custom-dial&quot;]{
	background-color: #f2f2f2;
}
QLabel#lblDialogTitle{
	font: 18px 'Montserrat SemiBold';
}


QFrame#frameDialog{
	background-color: transparent; 
	border-image: url(:/images/images/Popup_white.svg);
}

QPushButton[accessibleName=&quot;evo-custom-dial&quot;]
{
	border: 0px;
	border-radius: 15px;
	background-color: #324555;
	font: 16px 'Montserrat SemiBold';
	color: #F2F2F2;
}

QPushButton#btnDialogClosing{
	background-color: #F2F2F2;
}
QPushButton::hover[accessibleName=&quot;evo-custom-dial&quot;]
{
	background-color: #6b757d;

}

QPushButton::pressed[accessibleName=&quot;evo-custom-dial&quot;]
{
	background-color: #06bbb9;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frameDialog">
     <property name="accessibleName">
      <string>evo-custom-dial</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="accessibleName">
         <string>evo-custom-dial</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lblDialogTitle">
               <property name="font">
                <font>
                 <family>Montserrat SemiBold</family>
                 <pointsize>-1</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="accessibleName">
                <string>evo-custom-dial</string>
               </property>
               <property name="text">
                <string>Title</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnDialogClosing">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>evo-custom-dial</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>:/images/images/exit_button_black.png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lblDialogContent">
             <property name="text">
              <string>1. Ensure your gloves are 
 charged and powered on. 
2. Select a workplace and 
 a person ID 
3.Press the play to record data 
 and the pause button to end 
 a recording.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDialogCancel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="accessibleName">
           <string>evo-custom-dial</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDialogOk">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="accessibleName">
           <string>evo-custom-dial</string>
          </property>
          <property name="text">
           <string>End Application</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnDialogOk</sender>
   <signal>clicked()</signal>
   <receiver>CustomDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDialogCancel</sender>
   <signal>clicked()</signal>
   <receiver>CustomDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
